velox/common/base/Counters.h

```diff
@@ -248,6 +248,9 @@ constexpr folly::StringPiece kMetricSsdCacheGrowFileErrors{
 constexpr folly::StringPiece kMetricSsdCacheWriteSsdErrors{
     "velox.ssd_cache_write_ssd_errors"};
 
+constexpr folly::StringPiece kMetricSsdCacheWriteDropped{
```

Comment:
To align with other names (e.g. ..._write_ssd_errors), shall we call it kMetricSsdCacheWriteSsdDropped? So is the literal string and other places